src/test/ui/regions/regions-return-ref-to-upvar-issue-17403.stderr

```diff
@@ -1,9 +1,13 @@
 error: captured variable cannot escape `FnMut` closure body
   --> $DIR/regions-return-ref-to-upvar-issue-17403.rs:7:24
    |
+LL |         let mut x = 0;
+   |             ----- variable defined here
 LL |         let mut f = || &mut x;
-   |                      - ^^^^^^ returns a reference to a captured variable which escapes the closure body
-   |                      |
+   |                      - ^^^^^-
```

Comment:
in this specific case, the "variable used here" does't seem to be adding much, I wonder if we could suppress it

Reply:
I'm concerned that suppressing this could be confusing in a more complicated expression, e.g.:

```rust
let mut f = || some_expr(&mut x, &mut y).field // `field` derived from `&mut x`
```

In this example, I think it would make sense to annotate `some_expr(&mut x, &mut y).field` and `&mut x` separately.

Reply:
OK.